lib/runner.nix

```
@@ -72,7 +72,9 @@ pkgs.buildPackages.runCommand "microvm-${microvmConfig.hypervisor}-${hostName}"
   '') (builtins.attrNames binScriptPkgs)}
 
   mkdir -p $out/share/microvm
+  ${lib.optionalString microvmConfig.systemSymlink ''
   ln -s ${toplevel} $out/share/microvm/system
+  ''}
```

I first had this like that 

```suggestion
  '' + lib.optionalString microvmConfig.systemSymlink ''
  ln -s ${toplevel} $out/share/microvm/system
  '' + ''
```
which caused first an infinite recursion and after removing that code I got `9p/virtiofs shares not implemented for Firecracker` 😂  because probably the strings are tried to be added to the output of runCommand, not the multiline string itself.